src/controllers/messagesController.ts

```diff
@@ -132,4 +135,29 @@ export default class MessagesController {
         .fold<IGetLegalMessageAttachmentResponse>(identity, identity)
         .run()
     );
+
+  public readonly upsertMessageStatus = (
+    req: express.Request
+  ): Promise<
+    | IResponseErrorInternal
+    | IResponseErrorValidation
+    | IResponseErrorNotFound
+    | IResponseErrorTooManyRequests
+    | IResponseSuccessJson<MessageStatus>
```

Comment:
Why the return type of this API is not `MessageStatusAttributes`? I think that the return type of an API should return the information that was modified, to avoid another `GET` api call after the edit process.
Nevertheless the `MessageStatus` has information that are not useful for the client.

Reply:
You're right, I'm returning the wrong type. 
Specs define `SuccessResponse` as response in case of 200 
https://github.com/pagopa/io-backend/blob/321a46e5fcf6bb25ebcebc23e42ec58ca507c39b/api_backend.yaml#L290

Reply:
Anyway, openapi defined in functions-app-messages are returning just the `MessageStatus`
 https://github.com/pagopa/io-functions-app-messages/blob/c63eefcdeae8ac5c025e1e2116785eabdb2326a5/openapi/index.yaml#L160

Reply:
Even there the type diverge from the example, is an error?
```
examples:
  application/json: |
  status: ACCEPTED,
  version: 1,
  updated_at: 2022-02-28T16:35:15.993Z,
  is_read: true,
  is_archived: false
```

Reply:
Thank you. We refactored return type based on your comments in [a8dfc73](https://github.com/pagopa/io-backend/pull/879/commits/a8dfc73a34e3432c3cdb821f619bb72d367ded72) and [b7f43c1](https://github.com/pagopa/io-backend/pull/879/commits/b7f43c1d6febcd0895c449fc0dafd53d80894f67)